iceberg/iceberg-handler/src/test/queries/positive/dynamic_partition_pruning.q

```diff
@@ -0,0 +1,161 @@
+--! qt:dataset:srcpart
+SET hive.vectorized.execution.enabled=false;
```

Comment:
Do we need to turn this off? Will DPP work with vectorization?

Reply:
It is working - this part left here because the original patch was before vectorization was enabled.
Removed the line.
Thanks, Peter